.github/workflows/dev-linux.yml

```diff
@@ -52,6 +52,8 @@ jobs:
   build_musl:
     timeout-minutes: 30
     name: build_${{ matrix.arch }}_musl
+    env:
+      je_cv_pthread_getname_np: no
```

Comment:
How about adding a link to explain what does this mean?

Reply:
@Xuanwo  it works & some docs have been added, PTAL.